tests/contrib/pyramid/test_pyramid.py

```diff
@@ -263,3 +266,30 @@ def test_tween_overridden(self):
         self.app.get('/json', status=200)
         spans = self.tracer.writer.pop()
         eq_(len(spans), 0)
+
+
+class TestPyramidDistributedTracing(PyramidBase):
+    instrument = True
+
+    def get_settings(self):
+        return {
+            'datadog_distributed_tracing': True,
+        }
+
+    def test_distributed_tracing(self):
```

Comment:
all other tests run without distributed tracing so I think it's enough to keep only this one